frontend/src/components/samples/SamplesDetailContent.js

```diff
@@ -196,6 +210,26 @@ const SamplesDetailContent = ({samplesByID, sampleKindsByID, containersByID, ind
           </div>
         </Col>
       </Row>
+
+       <Title level={2} style={{ marginTop: '1em' }}>Processes</Title>
```

Comment:
i would put this above the version section if this cannot go side by side with it (i agree with the argument of the monitor size but this is starting to be a long page ;) ) ... I think this information is of more interest for the lab during day to day operation than the versions... The version section could also push this down and people may not realize it is there.